Jenkinsfile

```diff
@@ -56,6 +57,11 @@ pipeline {
                         }
                     }
                 }
+                stage ("Build Docs") {
+                   steps {
+                       sh 'TOXDIR=/tmp/$(basename ${WORKSPACE})/tox-docs make docs'
```

Comment:
I note this is the same line as Sam picked up here (https://github.com/bbc/rd-apmm-python-lib-mediatimestamp/pull/21#pullrequestreview-193076704) but should be resolved by a successful outcome in https://github.com/bbc/rd-ansible-ips-jenkins-slave/pull/9